src/locale/locales/fr/messages.po

```diff
@@ -4773,7 +4753,7 @@ msgstr "Rechercher tous les posts avec le mot-clé {displayTag}"
 
 #: src/screens/StarterPack/Wizard/index.tsx:467
 msgid "Search for feeds that you want to suggest to others."
-msgstr ""
+msgstr "Recherchez les fils d’actu que vous voulez suggérer à d’autres personnes."
```

Comment:
Here, I think that `search for feeds` could be read `search for SOME feeds` more that `search for THE feeds`, so let's use `des` instead of `les`.

```suggestion
msgstr "Recherchez des fils d’actu que vous voulez suggérer à d’autres personnes."
```

Reply:
SGTM